src/Calculator/Views/DateCalculator.xaml.cpp

```diff
@@ -164,7 +164,7 @@ void DateCalculator::CloseCalendarFlyout()
 {
     if (DateDiff_FromDate->IsCalendarOpen)
     {
-        DateDiff_FromDate->IsCalendarOpen = false;
+        DateDiff_FromDate->IsCalendarOpen = false;        
```

Comment:
nit: remove the extra spaces

Reply:
Updated

Reply:
An interesting discrepancy I found. In visual studio this didn't show as a pending change, but when I went to commit via git desktop, then it showed up as a change. Any idea why?

Reply:
weird, no idea!